### DISCUSSION TITLE
Bugs for the aluvm-0.11-beta2

### DISCUSSION THREAD

**Discussion by cymqqqq** [RGB-WG/rgb#87] [Q&A] [answered]
Hi there, I compiled the rgb-schemata 0.11-beta3 with the aluvm-0.11.beta2, then the bugs prints out:
<img width="849" alt="image" src="https://github.com/RGB-WG/rgb/assets/8555534/592a7695-c616-499d-9d98-c534ee3e7e48">

**Comment by dr-orlovsky** [comment, marked as answer]
Use the latest master. Do `cargo update`.
